crates/fortitude/src/check.rs

```diff
@@ -137,20 +135,13 @@ enum CheckStatus {
 }
 
 /// Check all files, report issues found, and return error code.
-pub fn check(args: CheckCommand, global_options: &GlobalConfigArgs) -> Result<ExitCode> {
-    let (cli, config_arguments) = args.partition()?;
-
-    // First we need to find and read any config file
-    let project_root = configuration::project_root(path_absolutize::path_dedot::CWD.as_path())?;
-    let file_configuration = Configuration::from_options(
-        parse_config_file(&global_options.config_file)?,
-        &project_root,
-    );
+pub fn check(args: CheckCommand, global_options: GlobalConfigArgs) -> Result<ExitCode> {
+    let (cli, config_arguments) = args.partition(global_options)?;
 
-    // Now, we can override settings from the config file with options
-    // from the CLI
-    let config = config_arguments.transform(file_configuration);
-    let settings = config.into_settings(&project_root)?;
+    // Construct the "default" settings. These are used when no `fortitude.toml`
+    // files are present, or files are injected from outside of the hierarchy.
+    let file_configuration = resolve::resolve(&config_arguments, cli.stdin_filename.as_deref())?;
+    let settings = file_configuration.settings;
```

Comment:
Unless I'm misunderstanding, I don't think this comment is accurate, as `resolve(...)` does look for a `fortitude.toml` relative to the cwd.